### ISSUE TITLE
Changing characterization factors not working

### ISSUE THREAD

**Issue by roeldegens** [LCA-ActivityBrowser/activity-browser#1257]
### Updating AB

- [X] Yes, I have [updated AB](https://github.com/LCA-ActivityBrowser/activity-browser#updating-the-ab) and still experience this issue

### What happened?

I tried adding a new characterization factor to an impact category. Adding the bisophere flow to the method is not an issue. However, I cannot change the amount and it is stuck at 0. If I double click the amount and change the number it resets back to 0. 
This is only a problem for biosphere flows that I added to the method. Biosphere flows that were already included are not an issue.

There is no message/error popping up in the anaconda prompt either. 

Exact details of what I did:

1. I copied the "Crustal scarcity indicator 2020 method. 
2. Than I added the following biosphere flow: Neodymium; natural resource, in ground; natural resource
3. I tried to change the number, however, this did not work and is stuck at 0

### Operating system

Windows 10

### Conda environment

```shell
activity-browser          2.9.7              pyhd8ed1ab_0    conda-forge
anyio                     3.5.0            py39haa95532_0
appdirs                   1.4.4              pyh9f0ad1d_0    conda-forge
argon2-cffi               21.3.0             pyhd3eb1b0_0
argon2-cffi-bindings      21.2.0           py39h2bbff1b_0
arrow                     1.2.3              pyhd8ed1ab_0    conda-forge
asteval                   0.9.29             pyhd8ed1ab_0    conda-forge
asttokens                 2.0.5              pyhd3eb1b0_0
astunparse                1.6.3              pyhd8ed1ab_0    conda-forge
attrs                     22.1.0           py39haa95532_0
backcall                  0.2.0              pyhd3eb1b0_0
beautifulsoup4            4.12.2             pyha770c72_0    conda-forge
bleach                    4.1.0              pyhd3eb1b0_0
brightway2                2.4.4              pyhd8ed1ab_0    conda-forge
brotli                    1.0.9                hcfcfb64_8    conda-forge
brotli-bin                1.0.9                hcfcfb64_8    conda-forge
brotli-python             1.0.9            py39h99910a6_8    conda-forge
brotlicffi                1.0.9.2          py39h99910a6_3    conda-forge
brotlipy                  0.7.0           py39ha55989b_1005    conda-forge
bw2analyzer               0.10               pyhd8ed1ab_0    conda-forge
bw2calc                   1.8.2            py39hcbf5309_0    conda-forge
bw2data                   3.6.6              pyhd8ed1ab_0    conda-forge
bw2io                     0.8.10             pyhd8ed1ab_0    conda-forge
bw2parameters             0.7                pyhd8ed1ab_0    conda-forge
bw_migrations             0.2                pyhd8ed1ab_0    conda-forge
bzip2                     1.0.8                h8ffe710_4    conda-forge
ca-certificates           2024.3.11            haa95532_0
certifi                   2024.2.2         py39haa95532_0
cffi                      1.15.1           py39h68f70e3_3    conda-forge
charset-normalizer        3.1.0              pyhd8ed1ab_0    conda-forge
click                     8.1.3           win_pyhd8ed1ab_2    conda-forge
colorama                  0.4.6              pyhd8ed1ab_0    conda-forge
comm                      0.1.2            py39haa95532_0
contourpy                 1.0.7            py39h1f6ef14_0    conda-forge
cryptography              40.0.2           py39hb6bd5e6_0    conda-forge
cycler                    0.11.0             pyhd8ed1ab_0    conda-forge
debugpy                   1.5.1            py39hd77b12b_0
decorator                 5.1.1              pyhd3eb1b0_0
defusedxml                0.7.1              pyhd3eb1b0_0
dill                      0.3.6              pyhd8ed1ab_1    conda-forge
docopt                    0.6.2                      py_1    conda-forge
eidl                      2.0.1              pyhd8ed1ab_0    conda-forge
eight                     1.0.1              pyhd8ed1ab_4    conda-forge
entrypoints               0.4              py39haa95532_0
et_xmlfile                1.1.0              pyhd8ed1ab_0    conda-forge
executing                 0.8.3              pyhd3eb1b0_0
fasteners                 0.17.3             pyhd8ed1ab_0    conda-forge
flask                     2.2.3              pyhd8ed1ab_0    conda-forge
fonttools                 4.39.3           py39ha55989b_0    conda-forge
freetype                  2.12.1               h546665d_1    conda-forge
future                    0.18.3             pyhd8ed1ab_0    conda-forge
gettext                   0.21.1               h5728263_0    conda-forge
glib                      2.76.1               h12be248_0    conda-forge
glib-tools                2.76.1               h12be248_0    conda-forge
gst-plugins-base          1.22.0               h001b923_2    conda-forge
gstreamer                 1.22.0               h6b5321d_2    conda-forge
icu                       72.1                 h63175ca_0    conda-forge
idna                      3.4                pyhd8ed1ab_0    conda-forge
importlib-metadata        6.4.1              pyha770c72_0    conda-forge
importlib-resources       5.12.0             pyhd8ed1ab_0    conda-forge
importlib_metadata        6.4.1                hd8ed1ab_0    conda-forge
importlib_resources       5.12.0             pyhd8ed1ab_0    conda-forge
inflate64                 0.3.1            py39ha55989b_3    conda-forge
intel-openmp              2023.0.0         h57928b3_25922    conda-forge
ipykernel                 6.19.2           py39hd4e2768_0
ipython                   8.12.0           py39haa95532_0
ipython_genutils          0.2.0              pyhd3eb1b0_1
itsdangerous              2.1.2              pyhd8ed1ab_0    conda-forge
jedi                      0.18.1           py39haa95532_1
jinja2                    3.1.2              pyhd8ed1ab_1    conda-forge
jsonschema                4.17.3           py39haa95532_0
jupyter_client            8.1.0            py39haa95532_0
jupyter_core              5.3.0            py39haa95532_0
jupyter_server            1.23.4           py39haa95532_0
jupyterlab_pygments       0.1.2                      py_0
kiwisolver                1.4.4            py39h1f6ef14_1    conda-forge
krb5                      1.20.1               heb0366b_0    conda-forge
lcms2                     2.15                 h3e3b177_1    conda-forge
lerc                      4.0.0                h63175ca_0    conda-forge
libblas                   3.9.0              16_win64_mkl    conda-forge
libbrotlicommon           1.0.9                hcfcfb64_8    conda-forge
libbrotlidec              1.0.9                hcfcfb64_8    conda-forge
libbrotlienc              1.0.9                hcfcfb64_8    conda-forge
libcblas                  3.9.0              16_win64_mkl    conda-forge
libclang                  16.0.1          default_h8b4101f_0    conda-forge
libclang13                16.0.1          default_h45d3cf4_0    conda-forge
libdeflate                1.18                 hcfcfb64_0    conda-forge
libffi                    3.4.2                h8ffe710_5    conda-forge
libglib                   2.76.1               he8f3873_0    conda-forge
libhwloc                  2.9.1                h51c2c0f_0    conda-forge
libiconv                  1.17                 h8ffe710_0    conda-forge
libjpeg-turbo             2.1.5.1              hcfcfb64_0    conda-forge
liblapack                 3.9.0              16_win64_mkl    conda-forge
libogg                    1.3.4                h8ffe710_1    conda-forge
libpng                    1.6.39               h19919ed_0    conda-forge
libsodium                 1.0.18               h62dcd97_0
libsqlite                 3.40.0               hcfcfb64_0    conda-forge
libtiff                   4.5.0                h6c8260b_6    conda-forge
libvorbis                 1.3.7                h0e60522_0    conda-forge
libwebp                   1.3.0                hcfcfb64_0    conda-forge
libwebp-base              1.3.0                hcfcfb64_0    conda-forge
libxcb                    1.13              hcd874cb_1004    conda-forge
libxml2                   2.10.4               hc3477c8_0    conda-forge
libxslt                   1.1.37               h0192164_0    conda-forge
libzlib                   1.2.13               hcfcfb64_4    conda-forge
lxml                      4.9.2            py39h0942119_0    conda-forge
m2w64-gcc-libgfortran     5.3.0                         6    conda-forge
m2w64-gcc-libs            5.3.0                         7    conda-forge
m2w64-gcc-libs-core       5.3.0                         7    conda-forge
m2w64-gmp                 6.1.0                         2    conda-forge
m2w64-libwinpthread-git   5.0.0.4634.697f757               2    conda-forge
markupsafe                2.1.2            py39ha55989b_0    conda-forge
matplotlib-base           3.7.1            py39haf65ace_0    conda-forge
matplotlib-inline         0.1.6            py39haa95532_0
mistune                   0.8.4           py39h2bbff1b_1000
mkl                       2022.1.0           h6a75c08_874    conda-forge
mrio_common_metadata      0.2.1              pyhd8ed1ab_0    conda-forge
msys2-conda-epoch         20160418                      1    conda-forge
multiprocess              0.70.14          py39ha55989b_3    conda-forge
multivolumefile           0.2.3              pyhd8ed1ab_0    conda-forge
munkres                   1.1.4              pyh9f0ad1d_0    conda-forge
nbclassic                 0.5.5            py39haa95532_0
nbclient                  0.5.13           py39haa95532_0
nbconvert                 6.5.4            py39haa95532_0
nbformat                  5.7.0            py39haa95532_0
nest-asyncio              1.5.6            py39haa95532_0
networkx                  3.1                pyhd8ed1ab_0    conda-forge
notebook                  6.5.4            py39haa95532_0
notebook-shim             0.2.2            py39haa95532_0
numpy                     1.23.5           py39hbccbffa_0    conda-forge
openjpeg                  2.5.0                ha2aaf27_2    conda-forge
openpyxl                  3.1.1            py39ha55989b_0    conda-forge
openssl                   3.1.0                hcfcfb64_0    conda-forge
packaging                 23.1               pyhd8ed1ab_0    conda-forge
pandas                    2.0.0            py39h2ba5b7c_0    conda-forge
pandocfilters             1.5.0              pyhd3eb1b0_0
parso                     0.8.3              pyhd3eb1b0_0
pathos                    0.3.0              pyhd8ed1ab_0    conda-forge
patsy                     0.5.3              pyhd8ed1ab_0    conda-forge
pcre2                     10.40                h17e33f8_0    conda-forge
peewee                    3.16.0           py39habf9222_0    conda-forge
pickleshare               0.7.5           pyhd3eb1b0_1003
pillow                    9.5.0            py39haa1d754_0    conda-forge
pint                      0.21               pyhd8ed1ab_0    conda-forge
pip                       23.1               pyhd8ed1ab_0    conda-forge
platformdirs              3.2.0              pyhd8ed1ab_0    conda-forge
pooch                     1.7.0              pyha770c72_3    conda-forge
pox                       0.3.2              pyhd8ed1ab_0    conda-forge
ppft                      1.7.6.6            pyhd8ed1ab_0    conda-forge
prometheus_client         0.14.1           py39haa95532_0
prompt-toolkit            3.0.36           py39haa95532_0
psutil                    5.9.5            py39ha55989b_0    conda-forge
pthread-stubs             0.4               hcd874cb_1001    conda-forge
pthreads-win32            2.9.1                hfa6e2cd_3    conda-forge
pure_eval                 0.2.2              pyhd3eb1b0_0
py7zr                     0.20.5             pyhd8ed1ab_0    conda-forge
pybcj                     1.0.1            py39ha55989b_2    conda-forge
pybcpy                    0.0.17             pyhd8ed1ab_0    conda-forge
pycparser                 2.21               pyhd8ed1ab_0    conda-forge
pycryptodomex             3.16.0           py39ha55989b_0    conda-forge
pygments                  2.11.2             pyhd3eb1b0_0
pyopenssl                 23.1.1             pyhd8ed1ab_0    conda-forge
pypardiso                 0.4.2              pyhd8ed1ab_1    conda-forge
pyparsing                 3.0.9              pyhd8ed1ab_0    conda-forge
pyperclip                 1.8.2              pyhd8ed1ab_2    conda-forge
pyppmd                    1.0.0            py39h99910a6_1    conda-forge
pyprind                   2.11.2          py39hcbf5309_1004    conda-forge
pyrsistent                0.18.0           py39h196d8e1_0
pyside2                   5.15.8           py39h56cbfbc_2    conda-forge
pysocks                   1.7.1              pyh0701188_6    conda-forge
python                    3.9.16          h4de0772_0_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python-fastjsonschema     2.16.2           py39haa95532_0
python-tzdata             2023.3             pyhd8ed1ab_0    conda-forge
python_abi                3.9                      3_cp39    conda-forge
pytz                      2023.3             pyhd8ed1ab_0    conda-forge
pywin32                   305              py39h2bbff1b_0
pywinpty                  2.0.10           py39h5da7b33_0
pyxlsb                    1.0.10             pyhd8ed1ab_0    conda-forge
pyzmq                     23.2.0           py39hd77b12b_0
pyzstd                    0.15.4           py39h95af829_0    conda-forge
qt-main                   5.15.8               h7f2b912_9    conda-forge
qt-webengine              5.15.8               h5b1ea0b_0    conda-forge
requests                  2.28.2             pyhd8ed1ab_1    conda-forge
salib                     1.4.7              pyhd8ed1ab_0    conda-forge
scipy                     1.10.1           py39hfbf2dce_0    conda-forge
seaborn                   0.12.2               hd8ed1ab_0    conda-forge
seaborn-base              0.12.2             pyhd8ed1ab_0    conda-forge
send2trash                1.8.0              pyhd3eb1b0_1
setuptools                67.6.1             pyhd8ed1ab_0    conda-forge
setuptools-scm            7.1.0              pyhd8ed1ab_0    conda-forge
six                       1.16.0             pyh6c4a22f_0    conda-forge
sniffio                   1.2.0            py39haa95532_1
soupsieve                 2.3.2.post1        pyhd8ed1ab_0    conda-forge
stack_data                0.2.0              pyhd3eb1b0_0
stats_arrays              0.6.5              pyhd8ed1ab_0    conda-forge
statsmodels               0.13.5           py39hc266a54_2    conda-forge
tabulate                  0.9.0              pyhd8ed1ab_1    conda-forge
tbb                       2021.9.0             h91493d7_0    conda-forge
terminado                 0.17.1           py39haa95532_0
texttable                 1.6.7              pyhd8ed1ab_0    conda-forge
tinycss2                  1.2.1            py39haa95532_0
tk                        8.6.12               h8ffe710_0    conda-forge
tomli                     2.0.1              pyhd8ed1ab_0    conda-forge
tornado                   6.2              py39h2bbff1b_0
tqdm                      4.65.0           py39hd4e2768_0
traitlets                 5.7.1            py39haa95532_0
typing-extensions         4.5.0                hd8ed1ab_0    conda-forge
typing_extensions         4.5.0              pyha770c72_0    conda-forge
tzdata                    2023c                h71feb2d_0    conda-forge
ucrt                      10.0.22621.0         h57928b3_0    conda-forge
unicodecsv                0.14.1                     py_1    conda-forge
unicodedata2              15.0.0           py39ha55989b_0    conda-forge
unidecode                 1.3.6              pyhd8ed1ab_0    conda-forge
urllib3                   1.26.15            pyhd8ed1ab_0    conda-forge
vc                        14.3                hb6edc58_10    conda-forge
voluptuous                0.13.1             pyhd8ed1ab_0    conda-forge
vs2015_runtime            14.34.31931         h4c5c07a_10    conda-forge
wcwidth                   0.2.5              pyhd3eb1b0_0
webencodings              0.5.1            py39haa95532_1
websocket-client          0.58.0           py39haa95532_4
werkzeug                  2.2.3              pyhd8ed1ab_0    conda-forge
wheel                     0.40.0             pyhd8ed1ab_0    conda-forge
whoosh                    2.7.4            py39hcbf5309_7    conda-forge
win_inet_pton             1.1.0              pyhd8ed1ab_6    conda-forge
winpty                    0.4.3                         4
wrapt                     1.15.0           py39ha55989b_0    conda-forge
xlrd                      2.0.1              pyhd8ed1ab_3    conda-forge
xlsxwriter                3.0.9              pyhd8ed1ab_0    conda-forge
xorg-libxau               1.0.9                hcd874cb_0    conda-forge
xorg-libxdmcp             1.1.3                hcd874cb_0    conda-forge
xz                        5.2.6                h8d14728_0    conda-forge
zeromq                    4.3.4                hd77b12b_0
zipfile-deflate64         0.2.0            py39ha55989b_3    conda-forge
zipp                      3.15.0             pyhd8ed1ab_0    conda-forge
zstd                      1.5.2                h12be248_6    conda-forge
```

**Comment by roeldegens** [comment]
Hello,

I fixed the issue, apparently I added the same biosphere flow twice which was raising issues (whoops)...

Everything is working fine!!

**Comment by marc-vdm** [comment]
Thanks for opening the issue though! This shouldn't be allowed to happen.

I'm closing #1214 in favour of this so you'll be notified when this issue is fixed!

**Comment by marc-vdm** [comment]
Hi, This has been resolved in `2.10.3`. You will be notified when the new version releases here.
